gemfiles/linux/Gemfile

```diff
@@ -4,59 +4,62 @@ source "https://rubygems.org"
 
 git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
 
-gem "async-http", "0.52.4"
-gem "aws-partitions", "1.337.0"
-gem "aws-sdk-core", "3.102.1"
-gem "aws-sdk-kms", "1.35.0"
-gem "aws-sdk-s3", "1.72.0"
-gem "aws-sdk-sqs", "1.29.0"
-gem "aws-sigv4", "1.2.1"
+# core gems
+
+gem "rake"
 gem "bundler", "2.1.4"
 gem "cool.io", "1.6.0"
-gem "elasticsearch", "7.8.0"
-gem "fluent-plugin-elasticsearch", "4.1.1"
-gem "fluent-plugin-kafka", "0.13.0"
-gem "fluent-plugin-prometheus", "1.8.1"
-gem "fluent-plugin-prometheus_pushgateway", "0.0.2"
-gem "fluent-plugin-record-modifier", "2.1.0"
-gem "fluent-plugin-rewrite-tag-filter", "2.3.0"
-gem "fluent-plugin-s3", "1.3.3"
-gem "fluent-plugin-td", "1.1.0"
-gem "fluent-plugin-webhdfs", "1.2.5"
+gem "sigdump", "0.2.4"
 gem "http_parser.rb", "0.6.0"
-gem "httpclient", "2.8.2.4"
-gem "jmespath", "1.4.0"
+gem "yajl-ruby", "1.4.1"
+gem "serverengine", '2.2.1'
 gem "msgpack", "1.3.3"
 gem "oj", "3.10.6"
+gem "tzinfo", "2.0.2"
+gem "tzinfo-data", "1.2020.1"
+gem "async-http", "0.52.4"
+
+# plugin gems
+
+gem "elasticsearch", "7.8.1"
+gem "fluent-plugin-elasticsearch", "4.1.1"
+gem "ruby-kafka", "1.2.0"
+gem "fluent-plugin-kafka", "0.14.0"
 gem "prometheus-client", "0.9.0"
-gem "rake"
-gem "ruby-kafka", "1.1.0"
-gem "serverengine", '2.2.1'
-gem "sigdump", "0.2.4"
+gem "fluent-plugin-prometheus", "1.8.2"
+gem "fluent-plugin-prometheus_pushgateway", "0.0.2"
+gem "jmespath", "1.4.0"
+gem "aws-partitions", "1.352.0"
+gem "aws-sdk-core", "3.104.3"
+gem "aws-sdk-kms", "1.36.0"
+gem "aws-sdk-sqs", "1.30.0"
+gem "aws-sigv4", "1.2.1"
+gem "aws-sdk-s3", "1.75.0"
+gem "fluent-plugin-s3", "1.4.0"
+gem "httpclient", "2.8.2.4"
 gem "td", "0.16.9"
 gem "td-client", "1.0.7"
-gem "tzinfo", "2.0.2"
-gem "tzinfo-data", "1.2020.1"
+gem "fluent-plugin-td", "1.1.0"
 gem "webhdfs", "0.9.0"
-gem "yajl-ruby", "1.4.1"
+gem "fluent-plugin-webhdfs", "1.2.5"
+gem "fluent-plugin-record-modifier", "2.1.0"
+gem "fluent-plugin-rewrite-tag-filter", "2.3.0"
+gem "nokogiri", "1.10.10"
 
 if /mswin|mingw/ =~ RUBY_PLATFORM
-  gem "nokogiri", "1.10.10"
   gem "ffi", "1.13.1"
   gem "ffi-win32-extensions", "1.0.3"
-  gem "fluent-plugin-parser-winevt_xml", "0.2.2"
-  gem "fluent-plugin-windows-eventlog", "0.7.0"
   gem "win32-api", "1.9.1"
   gem "win32-event", "0.6.3"
-  gem "win32-eventlog", "0.6.7"
-  gem "win32-ipc", "0.6.1"
-  gem "win32-service", "0.8.3"
+  gem "win32-ipc", "0.7.0"
+  gem "win32-service", "2.1.5"
```

Comment:
Fluentd gempsec cannot handle these newer gems by bundler:
https://github.com/fluent/fluentd/blob/master/fluentd.gemspec#L35-L36
If we should bump up version for these gems, we should upgrade dependent gem versions on Fluentd's gemspec. 
This causes package building error on Windows.